src/askvm/resources/list/__tests__/find.test.ts

```diff
@@ -28,7 +28,7 @@ function ask(code: string) {
 }
 
 test('find', async () => {
-  await expect(ask('call(f(1),)')).resolves.toBe(1);
+  await expect(ask('call(fun(1),)')).resolves.toBe(1);
```

Comment:
What made you change 'f' to 'fun' in AskCode?  
I thought your aim was to minimize the length of the code as much as possible?

Reply:
`f` now stands for fragment (block) which is more commonly used among resources. Aliases are still being figured out